src/components/project-card.vue

```diff
@@ -61,7 +61,8 @@
       status: '',
       loading: false,
       refreshInterval: null,
-      failedPipelines: []
+      failedPipelines: [],
+      audio: Config.root.linkToFailureSound
```

Comment:
Why do you extract a static config into `data`? :)

Reply:
Because without static declaration audio alerts doesn't work with FireFox browser

Reply:
But I still have issues with twice sound alerts on GC for old and new implementation, do you have any ideas on how I can fix it? and do you have the same issues in your environment?

Reply:
Already fixed) wrong browser settings) (about twice execution)

Reply:
> Because without static declaration audio alerts doesn't work with FireFox browser

This makes no sense to me. `this.audio` is just a string, isn't it?

Reply:
Yeap, it is. But in case, async audio play I have `DOM uncached  exception`

Reply:
I'll check this in a few hours :)

Reply:
Thank you

Reply:
Maybe I should move sound alert logic to refLoop?